wgpu/src/backend/web.rs

```diff
@@ -3462,6 +3467,57 @@ impl QueueWriteBuffer for WebQueueWriteBuffer {
     }
 }
 
+#[derive(Debug)]
+pub struct WebBuffer {
```

Comment:
This should be documented with the motivation on why this struct is needed instead of just using the web_sys::GpuBuffer (essentially what you wrote in the PR description)